### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rictic**
This change is [Reviewable](https://reviewable.io/reviews/polymer/polymer-editor-service/97)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,14 @@ Static analysis for web components in your text editor of choice!
...
 
-### Protocol
+### Supporting a new editor.
 
-The `polymer-language-service` command speaks the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol) over stdin and stdout.
-
-`polymer-editor-server` and its nonstandard protocol are deprecated, and we'll be migrating the atom and sublime text plugins over to use the `polymer-language-service` soon. The differences are fairly minor and migration should be done by the end of November 2016.
+As long as the editor supports the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol), it should be easy to get it to work with the polymer editor service. When the editor service is installed via `npm` it exposes a binary named `polymer-editor-service`. This binary speaks the LSP over stdin/stdout, so you just need to point the editor or a plugin to it that adds LSP support to that binary, and tell it that the polymer editor service will handle html, css, js, and json files. See the wiki page for setting up sublime text support for an example: https://github.com/Polymer/polymer-editor-service/wiki/Sublime-Text
```
Comment (TimvdLippe) [inline README.md]: Introduce LSP as acronym in the link, such that it is clear what it means.

Comment (arthurevans) [reply README.md]: Agree, we'd usually use "Language Server Protocol (LSP)"

Comment (rictic) [reply README.md]: +1, PTAL

Comment (arthurevans) [reply README.md]: This sentence is long and a bit confusing: "This binary speaks the LSP over stdin/stdout, so you just need to point the editor or a plugin to it that adds LSP support to that binary, and tell it that the polymer editor service will handle html, css, js, and json files."

Maybe:

This binary speaks the LSP over stdin/stdout. 

If you have an editor that supports LSP, or a plugin that adds LSP support, you can configure it to use the Editor Service. You'll need to provide the following information:

-   The path to the `polymer-editor-service` binary.
-   A list of file types that that the service supports (in this case, HTML, CSS, JavaScript and JSON files).

Comment (rictic) [reply README.md]: Yeah, that much much clearer!

**Pull request merged**